### ISSUE TITLE
Nuxt integration example

### ISSUE THREAD

**Issue by marekalgoud** [Jinjiang/vue-a11y-utils#15]
**Is your feature request related to a problem? Please describe.**

I would like to use vue-a11y-utils in a nuxt project, but when I tried to intergrate it in a component, I've this following error : 

```
SyntaxError: Cannot use import statement outside a module
```

**Describe the solution you'd like**

Give an example of nuxt intergration (plugin example would be perfect). On my side I had to add this to the nuxt config to make it work on server side : 

```
build: {
    transpile: [ 'vue-a11y-utils']
  },
```

Thanks for your work !

**Comment by Jinjiang** [comment]
Hi, is that still not working? I tried an empty nuxt project with vue-a11y-utils and it worked well.

![image](https://user-images.githubusercontent.com/206848/196085951-54476bca-168d-4b1b-ab86-d06501bb8d5d.png)

![image](https://user-images.githubusercontent.com/206848/196086082-1e5f6cc0-eb58-4f0c-b101-f9cf5612adf0.png)

**Comment by Jinjiang** [comment]
Anyway, the newly released v0.8.0 supports both CJS and ESM formats. So this problem has been solved. Thanks.
